calico_kubernetes/tests/kube_plugin_test.py

```diff
@@ -416,16 +416,16 @@ def test_delete_docker_interface(self):
             ], any_order=True)
 
     def test_configure_profile(self):
-        with patch_object(self.plugin, '_datastore_client',
-                          autospec=True) as m_datastore_client, \
-                patch_object(self.plugin, '_get_namespace_tag',
-                             autospec=True) as m_get_namespace_tag, \
-                patch_object(self.plugin, '_get_pod_config',
-                             autospec=True) as m_get_pod_config, \
-                patch_object(self.plugin, '_apply_rules',
-                             autospec=True) as m_apply_rules, \
-                patch_object(self.plugin, '_apply_tags',
-                             autospec=True) as m_apply_tags:
+        with patch.object(self.plugin, '_datastore_client',
```

Comment:
As I pointed out to Alex -- you don't need a patch.object here, since you're creating a new plugin for each test case. Just do `m_datastore_client = MagicMock(); self.plugin._datastore_client = m_datastore_client`.

I don't think you should bother changing this throughout the file, but worth fixing up wherever you're making edits, since it reduces the indentation (and gains us precious characters on each line)